README.md

```diff
@@ -49,14 +49,15 @@ Check out all related information [here](GSSoC.md)
   - `create database mapbot;`
   - Verify creation of the database `show databases;`
 - Unzip the StanfordCoreNLP package in the repository and keep the file names `stanford-corenlp-x.x.x.jar` and `stanford-corenlp-x.x.x-models.jar` handy.
-- Add config.py file to .gitignore to avoid pushing changes made to config
 - Run `git rm --cached config.py`
-- Edit the `config.py` file with the corresponding values
-  - user = "root"
-  - password = <your_root_password>
-  - host = "localhost"
-  - database = "mapbot"
-  - key = <your_Google_Cloud_API_key>
+-Create a file `.env` in the repository
```

Comment:
Specify the location relative to the repository where the file needs to be created.

Reply:
done